pages/workspace/tracking-plan/publishing.mdx

```diff
@@ -293,7 +293,7 @@ Example payload <Link passHref href="/implementation/avo-tracking-plan-webhook">
 | :-------------- | :--------------- | :---------------------------------------------------------------------------------------- |
 | branchId        | id               | Branch id                                                                                 |
 | branchName      | string           | Branch name                                                                               |
-| integrationId   | string           | Id of the publish integration (conistently the same for this integration)                 |
+| integrationId   | string           | Id of the publish integration (consistently the same for this integration)                 |
 | integrationName | string           | Name of the publish integration (seen in Tracking Plan -> publishing -> 'IntegrationName' |
 | publishDate     | Date string      | Timestamp when the webhook was triggered                                                  |
-| publishMethod   | { "type": type } | Desctiprion of the publish method, type can be "Manual" and "BranchMerge"                 |
+| publishMethod   | { "type": type } | Description of the publish method, type can be "Manual" and "BranchMerge"                 |
```

Comment:
Love these typo fixes 🧹✨ Can we add the typo check to CI?

Reply:
Yes, it's there!

Reply:
Was not looking at the nested folders before 😅